Dockerfile.pyinstaller

```diff
@@ -1,17 +0,0 @@
-FROM danihodovic/pyinstaller-builder:latest
-
-ARG PYTHON_VERSION=3.9.2
-
-RUN pyenv install $PYTHON_VERSION && pyenv global $PYTHON_VERSION
-
-RUN pip install poetry
-
-WORKDIR /app/
-
-COPY ./pyproject.toml ./poetry.lock /app/
-
-RUN poetry install
-
-COPY . /app/
-
-RUN eval "$(pyenv init -)" && pyinstaller pyinstaller.spec
```

Comment:
Why was this removed? I used this Dockerfile to build a pyinstaller binary and copy it from the container.

Reply:
I'd recommend using for example GitHub actions instead for the build process since that can create repeatable builds directly into a GitHub release. If you prefer, I can revert the deletion of the extra `Dockerfile.pyinstaller`.

Reply:
Do you have sample code or can you open a PR for GH actions that generate the binary?

Reply:
Maybe just revert the Dockerfile change and merge? It'd be great to have the borgmatic cli compatibility fix.